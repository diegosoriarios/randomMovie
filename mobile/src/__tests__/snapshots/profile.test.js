import React from 'react';

import renderer from 'react-test-renderer';
import Profile from '../../pages/profile/index';

test('Profile renders correctly', () => {
  const tree = renderer.create(<Profile />).toJSON();
  expect(tree).toMatchSnapshot();
});
