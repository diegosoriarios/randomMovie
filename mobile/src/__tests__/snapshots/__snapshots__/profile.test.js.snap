// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Image
      source={
        Object {
          "uri": "https://i.pravatar.cc/300",
        }
      }
      style={
        Object {
          "borderRadius": 300,
          "height": 300,
          "width": 300,
        }
      }
    />
    <Text
      style={
        Object {
          "fontSize": 28,
          "fontWeight": "bold",
        }
      }
    >
      Diego
    </Text>
  </View>
  <RCTScrollView
    data={Object {}}
    disableVirtualization={false}
    getItem={[Function]}
    getItemCount={[Function]}
    horizontal={false}
    initialNumToRender={10}
    keyExtractor={[Function]}
    maxToRenderPerBatch={10}
    onContentSizeChange={[Function]}
    onEndReachedThreshold={2}
    onLayout={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    stickyHeaderIndices={Array []}
    style={
      Object {
        "flex": 2,
      }
    }
    updateCellsBatchingPeriod={50}
    viewabilityConfigCallbackPairs={Array []}
    windowSize={21}
  >
    <View />
  </RCTScrollView>
</View>
`;
